### PR TITLE
add debug log for fixes response

### PR DESCRIPTION
**Pull request by mikeurbanski1**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/common/bridgecrew/integration_features/features/fixes_integration.py [hunk 1]
```diff
@@ -78,6 +79,8 @@ def _get_fixes_for_file(self, filename, file_contents, failed_checks):
             error_message = extract_error_message(response)
             raise Exception(f'Get fixes request failed with response code {response.status_code}: {error_message}')
 
+        logging.debug(f'Response from fixes API: {response.content}')
+
         fixes = json.loads(response.content)
         return fixes[0]
```
Comment (schosterbarak) [inline checkov/common/bridgecrew/integration_features/features/fixes_integration.py:85]: add protection to fixes - it would even be better.

Comment (mikeurbanski1) [reply checkov/common/bridgecrew/integration_features/features/fixes_integration.py:89]: Done

**Review by elaygl — APPROVED** [review]
Looks good

File: checkov/common/bridgecrew/integration_features/features/fixes_integration.py (hunk 1 above)
Comment (mikeurbanski1) [reply checkov/common/bridgecrew/integration_features/features/fixes_integration.py:89]: Oops, now done

**Comment by elaygl** [comment]
Shouldn't Homebrew run only after the merge?
image

**Comment by mikeurbanski1** [comment]
> Shouldn't Homebrew run only after the merge?
> image

Looks like it was changed today https://github.com/bridgecrewio/checkov/commit/6f2e32080ab0f80ca362fa48a6e860583f9828fa#diff-bef3099379a9a034a0ee7c4eb0e6eb98cde146b03245e3cdb27157c36062c400

**Comment by schosterbarak** [comment]
woops. i'll fix the hombrew one

**Pull request merged**